### PR TITLE
Added Button styles

### PR DESCRIPTION
**Pull request by san-jay-14**
`<button class="btn login-btn">Login</button><br>
<button class="btn small-btn">Small</button><br>
  <button class="btn medium-btn">medium</button><br>
  <button class="btn large-btn">large</button>`

Screenshot 2024-01-05 234357

### CODE REVIEW

File: button.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    font-size: var(--button-font-size);
+    text-align: center;
+    text-decoration: none;
+    cursor: pointer;
+    border-radius: 10px;
+}
+
+
+.login-btn {
+    font-family: var(--font-family);
```
Comment (hariFED) [inline button.css]: As font-family was given in root no need to include in every div/btn.

File: button.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    font-weight: var(--font-normal);
+    display: inline-block;
+    padding: 13px 30px;
```
Comment (hariFED) [inline button.css]: Add a padding variable in the root.(Name it as --login-btn-padding)

**Review by hariFED — COMMENTED** [review]
Commit the changes.

**Comment by san-jay-14** [comment]
Committed the changes bro..

**Review by aravind-digitech — APPROVED** [review]
Looks good.

**Review by hariFED — APPROVED** [review]
Approved

**Pull request merged**